cl_sii/rut/__init__.py

```diff
@@ -118,11 +118,29 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return f"Rut('{self.canonical}')"
 
+    def __lt__(self, other: object) -> bool:
+        if isinstance(other, Rut):
+            return int(self.digits) < int(other.digits)
+        else:
+            return NotImplemented
+
+    def __le__(self, other: object) -> bool:
+        return self.__lt__(other) or self.__eq__(other)
+
     def __eq__(self, other: object) -> bool:
         if isinstance(other, Rut):
             return self.canonical == other.canonical
         return False
 
+    def __gt__(self, other: object) -> bool:
```

Comment:
Implementing `__lt__` and `__le__` is enough. Having `__gt__` and `__ge__` as well is unnecessary and may introduce very-difficult-to-debug inconsistencies

Reply:
Resolved.